release.sh

```diff
@@ -2,32 +2,61 @@
 
 set -e
 
-last_tag=$(git tag --sort=-creatordate | head -n 1)
+last_tag=$(git tag --sort=-creatordate| head -n 1)
```

Comment:
Please revert back to what it was.

```suggestion
last_tag=$(git tag --sort=-creatordate | head -n 1)
```